Apps/Log/Log.xcodeproj/project.pbxproj

```diff
@@ -498,6 +506,7 @@
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
+				DEVELOPMENT_ASSET_PATHS = DevResources;
```

Comment:
Hu de utalom a build settingset 😄 Irok magamnak egy todot hogy ezek menjenek ki xcconfigba.

Reply:
We might (we sure do) want to document that as well 🙄